libs/vault/src/components/copy-cipher-field.directive.ts

```
@@ -52,7 +50,7 @@ export class CopyCipherFieldDirective implements OnChanges {
   @HostListener("click")
   async copy() {
     const value = this.getValueToCopy();
-    await this.copyCipherFieldService.copy(value, this.action, this.cipher);
+    await this.copyCipherFieldService.copy(value ?? "", this.action, this.cipher);
```

❓ Should this throw an error rather than use a default value? I couldn't decide

---

I'm okay with what's currently there